packages/vuetify/src/labs/VDataTable/VDataTableVirtual.tsx

```diff
@@ -114,6 +116,7 @@ export const VDataTableVirtual = genericComponent<VDataTableVirtualSlots>()({
         class="v-data-table"
         style={{
           '--v-table-row-height': convertToUnit(itemHeight.value),
+          'v-data-table--loading': props.loading,
```

Comment:
It's a class, not a style

Reply:
Ugh you're right of course. I hope commit-amending and force-pushing is the correct way to go? To abide with the commit rules of the contributor guidelines.